.github/workflows/snapshot-firestore.yml

```diff
@@ -0,0 +1,21 @@
+name: Snapshot firestore
+on:
+  schedule:
+    # At 19:00 UTC / 7:00 PST on every day-of-week from Monday through Thursday.
+    - cron: '0 19 * * 1-4'
```

Comment:
Btw I think there's a way to set a GCS bucket rule where it deletes stuff after a certain amount of time. Do you want to add a rule to clean out this bucket after like 15 days so we don't have to remember to do it manually?

Reply:
I've set it to 90 days. Partially because the bucket itself is Nearline, which penalizes you if you delete before 30 days.

I've also changed this to just do a weekly backup on Mondays.